### ISSUE TITLE
First HTTP2 requests take more time

### ISSUE THREAD

**Issue by kaatt** [sindresorhus/got#1168]
#### Describe the bug

 - Node.js version: v13 / v12.11.1
 - OS & version: macOS / Linux 4.15.11

#### Actual behavior

```
> console.time('done');got('https://facebook.com').then(x => console.timeEnd('done'))
Promise { <pending> }
> done: 302.025ms
> console.time('done');got('https://facebook.com', { http2: true }).then(x => console.timeEnd('done'))
Promise { <pending> }
> done: 60188.968ms
```
60s for HTTP2 request compared to .3s for HTTP1 request

#### Expected behavior

HTTP/2 should be faster or around the same speed as HTTP/1

#### Checklist

- [x] I have read the documentation.
- [x] I have tried my code with the latest version of Node.js and Got.

Cross posted on https://github.com/szmarczak/http2-wrapper/issues/38

**Comment by szmarczak** [comment]
It may be due to https://github.com/szmarczak/http2-wrapper/issues/37

**Comment by szmarczak** [comment]
Fixed in 4e8de8e32cccaafddf0c6ef4f31716810ab4a14b

Sometimes the first request may take as long as the HTTP1 one, but generally all HTTP2 requests should be 2x faster. The `facebook.com` issue is a Node.js bug.

**Comment by duongvanba** [comment]
Hi, are there any way to tmp fix this facebook.com issue ?
